templates/vue/README.md

```diff
@@ -0,0 +1,101 @@
+near-blank-project
+==================
+
+This [Vue] app was initialized with [create-near-app]
+
+
+Quick Start
+===========
+
+To run this project locally:
+
+1. Prerequisites: Make sure you've installed [Node.js] ≥ 12
+2. Install dependencies: `npm install`
+3. Run the local development server: `npm run dev` (see `package.json` for a
+   full list of `scripts` you can run with `npm`)
+
+Now you'll have a local development environment backed by the NEAR TestNet!
+
+Go ahead and play with the app and the code. As you make code changes, the app will automatically reload.
+
+
+Exploring The Code
+==================
+
+1. The "backend" code lives in the `/contract` folder. See the README there for
+   more info.
+2. The frontend code lives in the `/src` folder. `/src/main.js` is a great
+   place to start exploring.
+3. Tests: there are different kinds of tests for the frontend and the smart
+   contract. See `contract/README` for info about how it's tested. The frontend
+   code gets tested with [jest]. You can run both of these at once with `npm
+   run test`.
```

Comment:
@luciotato does this look correct to you? Is there anything you would like to update or add to the NEAR+Vue-specific README?

Reply:
looks good to me